configure.ac

```diff
@@ -1585,7 +1605,8 @@ AM_CONDITIONAL([HTOP_DRAGONFLYBSD], [test "$my_htop_platform" = dragonflybsd])
 AM_CONDITIONAL([HTOP_NETBSD], [test "$my_htop_platform" = netbsd])
 AM_CONDITIONAL([HTOP_OPENBSD], [test "$my_htop_platform" = openbsd])
 AM_CONDITIONAL([HTOP_DARWIN], [test "$my_htop_platform" = darwin])
-AM_CONDITIONAL([HTOP_DARWIN_IOS6], [test "$htop_ios6_platform" = yes])
+AM_CONDITIONAL([HAVE_IOKIT], [test "x$htop_have_iokit" = xyes])
+AM_CONDITIONAL([HAVE_LIBPROC], [test "x$htop_have_libproc" = xyes])
```

Comment:
Use `AM_CONDITIONAL` sparingly. That macro is more useful for adding or removing what source files to compile within the `Makefile`. It's not meant for appending `AM_CFLAGS` or `AM_CPPFLAGS` or relevant build variables.

Reply:
Yeah. I just need one now.